README.md

```diff
@@ -133,7 +132,6 @@ Crossplane is under the Apache 2.0 license.
 <!-- Named links -->
 
 [Crossplane]: https://crossplane.io
-[Install Crossplane]: docs/getting-started/install-configure.md
```

Comment:
This link is used twice - it should stay in or it will break the link in the opening bullet point.